_includes/head.html

```diff
@@ -0,0 +1,16 @@
+<meta charset="utf-8">
+<title>{{ site.description }}</title>
+<meta name="description" content="{% if page.description %}{{ page.description }}{% else %}{{ site.description }}{% endif %}">
+
+<meta property="og:type" content="article">
+<meta property="og:title" content="{% if page.title %}{{ page.title }} &#8211; {% endif %}{{ site.title }}">
+<meta property="og:description" content="{% if page.description %}{{ page.description }}{% else %}{{ site.description }}{% endif %}">
+<meta property="og:url" content="{{ site.url }}{{ page.url }}">
+<meta property="og:site_name" content="{{ site.title }}">
+
+<meta http-equiv="content-type" content="text/html; charset=utf-8" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<link rel="stylesheet" href="{{ site.url }}/css/base.css" type="text/css" />
+<link rel="stylesheet" href="{{ site.url }}/css/github.min.css" type="text/css" />
+<link rel="stylesheet" href="{{ site.url }}/css/octicons.css" type="text/css" />
```

Comment:
@johnbrett Because this is deployed a subdir, the hrefs here are incorrect as they'll reference **nodeschool.github.io/css** instead of **nodeschool.github.io/dublin/css**. Won't be able to get around the fixing this til later, so maybe yourself or @davecocoa can do it if you have time before I do :+1: